addons/mail/static/src/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -12,7 +12,7 @@
                                 'o-active': chatter.isComposerVisible and chatter.composer and !chatter.composer.isLog,
                                 'o-bordered': chatter.hasExternalBorder,
                             }"
-                            t-att-disabled="chatter.isDisabled"
+                            t-att-disabled="chatter.isDisabled or (chatter.thread and !chatter.thread.hasWriteAccess)"
```

Comment:
Note that it is not correct. Post rights are also managed by the ``_mail_post_access`` parameter. I think there was already a fix done recently, could you check (maybe in 16) to backport it maybe ?